integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -556,29 +559,27 @@ public void testDomainIntrospectVersionRolling() {
           getPodCreationTime(introDomainNamespace, managedServerPodNamePrefix + i));
     }
 
-    logger.info("Getting port for default channel");
-    int adminServerPort = assertDoesNotThrow(()
-        -> getServicePort(introDomainNamespace, getExternalServicePodName(adminServerPodName), "default"),
-        "Getting admin server port failed");
-
-    // create a temporary WebLogic WLST property file
-    File wlstPropertiesFile = assertDoesNotThrow(() -> File.createTempFile("wlst", "properties"),
-        "Creating WLST properties file failed");
-    Properties p = new Properties();
-    p.setProperty("admin_host", adminServerPodName);
-    p.setProperty("admin_port", Integer.toString(adminServerPort));
-    p.setProperty("admin_username", ADMIN_USERNAME_DEFAULT);
-    p.setProperty("admin_password", ADMIN_PASSWORD_DEFAULT);
-    p.setProperty("cluster_name", clusterName);
-    p.setProperty("new_admin_port", Integer.toString(newAdminPort));
-    p.setProperty("test_name", "change_admin_port");
-    assertDoesNotThrow(() -> p.store(new FileOutputStream(wlstPropertiesFile), "wlst properties file"),
-        "Failed to write the WLST properties to file");
-
-    // change the admin server port to a different value to force pod restart
-    logger.info("changing the admin server port to a different value to force pod restart");
-    Path configScript = Paths.get(RESOURCE_DIR, "python-scripts", "introspect_version_script.py");
-    executeWLSTScript(configScript, wlstPropertiesFile.toPath(), introDomainNamespace);
+    //change admin port from 7001 to 7005
+    String restUrl = "/management/weblogic/latest/edit/servers/" + adminServerName;
+
+    String curlCmd = "curl -v"
+        + " -u " + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT
+        + " -H X-Requested-By:MyClient "
+        + " -H Accept:application/json "
+        + " -H Content-Type:application/json -d \"{listenPort: " + newAdminPort + "}\" "
+        + " -X POST http://" + adminServerPodName + ":7001" + restUrl;
+    logger.info("Command to set HTTP request and get HTTP response {0} ", curlCmd);
```

Comment:
Don't we have to commit the edit session ?

Reply:
Looks like REST does this transaction end to end(start edit session, change, commit)